packages/gestalt/src/Avatar.js

```diff
@@ -84,6 +84,12 @@ export default class Avatar extends React.PureComponent<AvatarProps, State> {
 
   handleImageError = () => this.setState({ isImageLoaded: false });
 
+  componentWillReceiveProps = (nextProps: *) => {
```

Comment:
This lifecycle method is deprecated, we shouldnt be using it to reset state https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

Reply:
Replaced by `componentDidUpdate` method